### PR TITLE
[WIP] Mantine v7

### LINKED ISSUE #45941: Mantine v7
Now that we're on React 18, we should upgrade to mantine v7 sooner rather than later to take advantage of new components, better component APIs, more performant styling, and better accessiblity.

see:
- https://github.com/metabase/metabase/pull/33368

```[tasklist]
### Tasks
- [x] Update Branch to Mantine v7
- [x] Setup Base Theme
- [ ] Convert each set of overrides to CSS
- [ ] Design review + figma comparison
- [ ] setup Loki tests against UI components storybook
```

```[tasklist]
### Follow-ups
- [ ] remove `theme.fn.themeColor` usage?
- [ ] audit all `styles` props: https://mantine.dev/guides/6x-to-7x/#styles-prop
- [ ] change spacing to gap props on stack components
- [ ] overrides truncation
```

### PR DESCRIPTION
**Pull request by rafpaf**
closes https://github.com/metabase/metabase/issues/45941

### CODE REVIEW

**Comment by rafpaf** [comment]
* **#48465** <a href="https://app.graphite.dev/github/pr/metabase/metabase/48465?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#48464** <a href="https://app.graphite.dev/github/pr/metabase/metabase/48464?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @rafpaf and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: frontend/src/metabase/browse/databases/BrowseDatabases.tsx [hunk 1]
```diff
@@ -52,9 +48,14 @@ export const BrowseDatabases = () => {
       <BrowseDataHeader />
       <BrowseMain>
         <BrowseSection>
-          <DatabaseGrid data-testid="database-browser">
+          <SimpleGrid
+            data-testid="database-browser"
+            cols={{ base: 1, md: 2, lg: 3 }}
+            spacing="md"
+            w="100%"
+          >
```
Comment (iethree) [inline frontend/src/metabase/browse/databases/BrowseDatabases.tsx:56]: 😍

File: frontend/src/metabase/lib/embed.js [hunk 2]
```diff
@@ -57,6 +57,18 @@ function sendMessage(message) {
   window.parent.postMessage({ metabase: message }, "*");
 }
 
+function isFitViewportMode() {
+  const root = document.getElementById("root");
+
+  // get the first div, which may be preceded by style nodes
+  const firstChild = root?.querySelector("div");
```
Comment (iethree) [inline frontend/src/metabase/lib/embed.js:64]: we should validate this behavior in embedding.

**Comment by iethree** [comment]
(closing this PR to get the benefit of some new testing tooling on another)

**Pull request closed without merging**